server/src/controllers/userController.js

```diff
@@ -157,7 +31,38 @@ const addUsers = (req, res) => {
   .catch((err) => res.send('Error adding user', err));  
 }
 
-// Triggred from 'PUT /slack/users' 
+// Triggered from 'POST /api/users/user' 
+// Creates new user or updates existing user's access token
+const addUser = (req, res) => {
+  let user = req.body.user;
+  let name = user.name;
+  let email = user.email;
+  let photo = user.photo;
+  let slackUserId = user.slackUserId;
+  let slackTeamId = user.slackTeamId;
+  let accessToken = user.accessToken;
+  let location = 'San Francisco';
```

Comment:
instead of defining here, we could set default in the model. This is a nitpick though